e3sm_diags/derivations/derivations.py

```diff
@@ -754,9 +894,18 @@
         ("surf_mom_flux_V",): lambda tauv: -tauv,  # EAMxx
     },
     "CLDICE": {("cli",): rename},
-    "TGCLDIWP": {("clivi",): rename},
-    "CLDLIQ": {("clw",): rename},
-    "TGCLDCWP": {("clwvi",): rename},
+    "TGCLDIWP": {
+        ("clivi",): rename,
+        ("IceWaterPath",): rename,  # EAMxx
+    },
+    "CLDLIQ": {
+        ("clw",): rename,
+        ("qc",): rename,  # EAMxx
+    },
+    "TGCLDCWP": {
+        ("clwvi",): rename,
+        ("LiqWaterPath",): rename,  # EAMxx Check if rain water is inlcuded?
```

Comment:
Does the question `Check if rain water is included?` still apply?

Reply:
@crterai could you confirm if `LiqWaterPath` and TGCLDCWP  are equivalent?

Reply:
Yes, they're equivalent